### PR TITLE
Bump jetty version from 9.2.28 to 9.4.28

### PR DESCRIPTION
**Pull request by bradyoo**
Vulnerability: https://github.com/advisories/GHSA-84q7-p226-4x5w

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/web/MasterWebServer.java [hunk 1]
```diff
@@ -97,7 +98,7 @@ public void init() throws ServletException {
         errorHandler.addErrorPage(404, "/");
         mServletContextHandler.setErrorHandler(errorHandler);
       }
-    } catch (MalformedURLException e) {
+    } catch (IOException e) {
```
Comment (bradyoo) [inline core/server/master/src/main/java/alluxio/web/MasterWebServer.java:101]: justification for this change:

Here is the implementation of the code that previously throwed MalformedURLException

  public static Resource newResource(String resource, boolean useCaches) throws IOException {
    URL url = null;

    try {
      url = new URL(resource);
    } catch (MalformedURLException var6) {
      if (!resource.startsWith("ftp:") && !resource.startsWith("file:") && !resource.startsWith("jar:")) {
        try {
          if (resource.startsWith("./")) {
            resource = resource.substring(2);
          }

          File file = (new File(resource)).getCanonicalFile();
          return new PathResource(file);
        } catch (IOException var5) {
          var5.addSuppressed(var6);
          throw var5;
        }
      }

      LOG.warn("Bad Resource: " + resource, new Object[0]);
      throw var6;
    }

**Comment by bradyoo** [comment]
alluxio-bot, test this please.

**Comment by bradyoo** [comment]
ping @gpang

File: core/server/master/src/main/java/alluxio/web/MasterWebServer.java (hunk 1 above)
Comment (gpang) [reply core/server/master/src/main/java/alluxio/web/MasterWebServer.java:100]: Are you saying now this code will throw other IOExceptions other than `MalformedURLException`?

**Comment by bradyoo** [comment]
@gpang See the comment I put on the other one where I changed from MalformedURLException to IOException. The underlying code no longer throws MalformedURLException. It only throws IOException.

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**